src/en/news/blog/2021/managing-grafana-dashboards-with-grafonnet/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Managing Grafana Dashboards with Grafonnet"
+date: "2021-09-17"
+author: "Aashish Sharma"
+---
+
+The process of maintaing grafana dashboards or updating the dashboard JSON files can sometimes become a mess as the size of the JSON files becomes too large to debug it. In this blog you can find out how grafana dashboards can be managed as code using Grafonnet.
+
+## About Grafonnet
+
+In case you already know what [Grafonnet](https://github.com/grafana/grafonnet-lib) is, it would be easier for you to get going with grafonnet. For those who don’t know: Grafonnet is a [Jsonnet](https://jsonnet.org) library – with Jsonnet being a JSON extension to create files using abstractions such as variables, functions, conditionals and more.Grafonnet consists of different functions we can use to create the JSON objects required to compose Grafana dashboards or panels. the resulting JSON files are then imported into grafana.
```

Comment:
I'd also mention that you explored the grafanalib Python library, and the pros/cons that moved us to go for this. You can also link the ceph-mixins project as a precedent of this effort (pure jsonnet vs. grafonnet).